### PR TITLE
fix deadlink

### PR DESCRIPTION
**Pull request by stewart-yu**
fix bad link. 

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6363)

### CODE REVIEW

File: cn/docs/admin/cluster-large.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ AWS使用的规格为：
 
 * 当扩大集群规模时，如果涉及，相应扩大以下插件的内存和CPU限制 (通过一个实例处理整个集群，因此其内存和CPU使用量往往与集群的大小/负载成比例增长)：
   * [InfluxDB 和 Grafana](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml)
-  * [kubedns, dnsmasq, 和 sidecar](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/kubedns-controller.yaml.in)
-  * [Kibana](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/fluentd-elasticsearch/kibana-controller.yaml)
+  * [kubedns, dnsmasq, 和 sidecar](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/kube-dns.yaml.in)
```
Comment (tengqm) [inline cn/docs/admin/cluster-large.md]: This should point to the directory instead of a single file

File: cn/docs/admin/cluster-large.md [hunk 2]
```diff
@@ -98,8 +98,8 @@ AWS使用的规格为：
 
 * 当扩大集群规模时，如果涉及，相应扩大以下插件的内存和CPU限制 (通过一个实例处理整个集群，因此其内存和CPU使用量往往与集群的大小/负载成比例增长)：
   * [InfluxDB 和 Grafana](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml)
-  * [kubedns, dnsmasq, 和 sidecar](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/kubedns-controller.yaml.in)
-  * [Kibana](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/fluentd-elasticsearch/kibana-controller.yaml)
+  * [kubedns, dnsmasq, 和 sidecar](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/kube-dns.yaml.in)
+  * [Kibana](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/fluentd-elasticsearch/kibana-deployment.yaml)
```
Comment (tengqm) [inline cn/docs/admin/cluster-large.md]: This too. Because it has both a deployment and a service to create.

**Comment by stewart-yu** [comment]
Done, thanks @tengqm

**Comment by stewart-yu** [comment]
cc @tengqm  Please help review this pr, thanks!

**Comment by tengqm** [comment]
This is not a valid change.
If wget https://k8s.io/docs/..../abc.yaml is not returning you a YAML, something is wrong.

**Comment by stewart-yu** [comment]
Results in mine VMs:
```shell
root@SZX1000380318:/ycj/tempfile# wget https://raw.githubusercontent.com/kubernetes/website/master/docs/tutorials/stateful-application/zookeeper.yaml
--2017-11-28 16:13:08--  https://raw.githubusercontent.com/kubernetes/website/master/docs/tutorials/stateful-application/zookeeper.yaml
Connecting to 10.229.53.140:8118... connected.
Proxy request sent, awaiting response... 200 OK
Length: 2967 (2.9K) [text/plain]
Saving to: ‘zookeeper.yaml’

100%[=====================================================================================>] 2,967       --.-K/s   in 0.001s  

2017-11-28 16:13:09 (5.54 MB/s) - ‘zookeeper.yaml’ saved [2967/2967]
```
That's ok! @tengqm

**Comment by tengqm** [comment]
@chenopis Please take a look at this. I have searched the whole project for the secret sauce but failed. It seems that files from /docs/tutorials/stateful-applications/ are suffering from the problem that raw yaml files cannot be returned.

e.g.  This is bad:
```
wget -v https://kubernetes.io/docs/tutorials/stateful-application/webp.yaml
```
But, this is good:
```
wget -v https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/local-volumes.yaml
```
This is good too:
```
wget -v https://kubernetes.io/docs/tutorials/clusters/hello-apparmor-pod.yaml
```

**Comment by tengqm** [comment]
related issue report: #6043

**Comment by chenopis** [comment]
I think using `https://raw.githubusercontent.com/kubernetes/website/master/...` is a sufficient workaround. I have a guess as to why this might be happening and will follow up in the issue.

**Comment by tengqm** [comment]
Okay, @chenopis. Let's get the workaround in and figure out a proper fix later by tracking #6043.

**Pull request merged**